src/main/java/app/bpartners/api/repository/jpa/InvoiceJpaRepository.java

```diff
@@ -26,6 +27,8 @@ List<HInvoice> findAllByIdUserAndArchiveStatusAndTitleContainingIgnoreCase(
 
   List<HInvoice> findByIdUserAndRef(String idAccount, String ref);
 
+  int countByIdUserAndCreatedDatetimeBetween(String idUser, Instant from, Instant to);
```

Comment:
No ! I already said it's not the createdDatetime that should be filtered, but sendingDate